### DISCUSSION TITLE
Syntax highlighting for html imbedded in JSON.

### DISCUSSION THREAD

**Discussion by aditr9** [microsoft/monaco-editor#3867] [Q&A]
So I am making a project that uses the monaco editor as its base text editor. If editor language is json, but for one of the fields, the value is html, I am wondering if it is possible to have syntax highlighting for the html aspect? I have tried to use monaco.languages.registerEmbeddedLanguages but it is not working.

**Comment by tamayika** [comment]
I don't think `monaco.languages.registerEmbeddedLanguages` function exists in monaco-editor.
